evalml/tests/component_tests/test_arima_regressor.py

```diff
@@ -269,3 +269,35 @@ def test_different_time_units_out_of_sample(
 
     assert (y_pred_sk.values == y_pred.values).all()
     assert y_pred.index.equals(X[15:].index)
+
+
+def test_arima_supports_boolean_features():
+    X = pd.DataFrame({"dates": pd.date_range("2021-01-01", periods=10)})
+    X.ww.init()
+    X.ww["bool_1"] = (
+        pd.Series([True, False])
+        .sample(n=10, replace=True, random_state=0)
+        .reset_index(drop=True)
+    )
+    X.ww["bool_2"] = (
+        pd.Series([True, False])
+        .sample(n=10, replace=True, random_state=1)
+        .reset_index(drop=True)
+    )
+    y = pd.Series(range(10))
+
+    ar = ARIMARegressor(time_index="dates")
+
+    ar._component_obj = MagicMock()
+    ar.fit(X, y)
+
+    pd.testing.assert_series_equal(
+        ar._component_obj.fit.call_args[1]["X"]["bool_1"], X["bool_1"].astype(float)
+    )
+    pd.testing.assert_series_equal(
+        ar._component_obj.fit.call_args[1]["X"]["bool_2"], X["bool_2"].astype(float)
+    )
+
+    ar = ARIMARegressor(time_index="dates")
```

Comment:
just wondering-- `ar._component_obj = MagicMock()` no longer applies to this line, right? Is that intentional? Are we just checking that we can actually fit/predict?

If so, should we also check that predict doesn't return nan values?

Reply:
Yea the intention is to check that we can call non-mocked fit/predict on data with bools and not error out. I will add a comment and check for non-nan!